static/js/public/snap-details/channelMap.js

```diff
@@ -145,14 +147,18 @@ function initOpenSnapButtons() {
       iframe.src = `snap://${name}`;
       document.body.appendChild(iframe);
 
-      if (typeof ga !== 'undefined') {
+      // Only send the QA event on first click, any subsequent clicks
+      // on this page can be assumed as the same intent.
+      if (typeof ga !== 'undefined' && attempted === false) {
```

Comment:
Any reason for strong comparison with `false`? Wouldn't `!attempted` be enough?

Reply:
not specifically I just like :muscle: things, do you think I should change it?

Reply:
I would say that `!attempted` reads better ("not attempted"). And `attempted === false` makes me think that there is some hidden reason why this needs to be `false` and nothing else. But maybe it's just me ;)